tests/setup.ts

```diff
@@ -1,6 +1,27 @@
 import '@testing-library/jest-dom/extend-expect';
+let windowSpy: jest.SpyInstance;
 
 if (global.window) {
-  const { getComputedStyle } = window;
-  window.getComputedStyle = (elt) => getComputedStyle(elt);
+  // need to mock this function to pass default values as 0 so that we can avoid NaN warning in some of components
```

Comment:
This solution will not address the warning for consumers of this library, thus it does not solve the issue.

Reply:
@p44dyoachim  This will fix the warnings in maifest tests, since you already have a draft PR which fixes that issue on consumer side as well, so i did not added those changes here.

If you want i can include the changes here, or we can merge your draft PR.
Do let me know how you wanna go about it ?